eloquent/distribution.yaml

```diff
@@ -1307,7 +1307,6 @@ repositories:
       url: https://github.com/ros2-gbp/ros_environment-release.git
       version: 2.4.0-1
     source:
-      test_pull_requests: true
```

Comment:
This seems to pass: http://build.ros2.org/view/Edev/job/Edev__ros_environment__ubuntu_bionic_amd64/6/